src/spaceObjects/asteroid.cpp

```diff
@@ -65,6 +65,13 @@ void Asteroid::drawOnRadar(sf::RenderTarget& window, sf::Vector2f position, floa
     window.draw(object_sprite);
 }
 
+std::unordered_map<string, string> Asteroid::getGMInfo()
+{
+    std::unordered_map<string, string> ret = SpaceObject::getGMInfo();
+    ret["typeName"] = "Asteroid";
```

Comment:
No need for these typeName, as the GM screen could just use the `getMultiplayerClassIdentifier` from the MultiplayerObject base class to get the type of any object.

Reply:
1. Simply `ret["typeName"] = getMultiplayerClassIdentifier()` for `SpaceObject`? (That would be what I was looking for.)

2. Any suggestion for "typeName"?